### DISCUSSION TITLE
Time Tracking on osTicket v1.18

### DISCUSSION THREAD

**Discussion by MaticSulc** [osTicket/osTicket#6598] [Q&A] [answered]
Hi,
I found the original PR and can't seem to find a working version of Time tracking on the latest stable version (that works on php 8.1).
We would need each ticket to have a time field + some reporting for each organization.

Regarding to this PR: https://github.com/osTicket/osTicket/pull/3231

Comment: https://github.com/osTicket/osTicket/pull/3231#issuecomment-228284503
has exactly what I would need :)

Thanks!

**Comment by JediKev** [comment, marked as answer]
@MaticSulc

This is the wrong place to ask such questions. You will need to use [our Community Supported Forum](https://forum.osticket.com) for any questions, assistance, guidance, etc.

Github is a place to report Issues (actual bugs), to submit Feature Requests, or to submit Pull Requests.

Cheers.
